superset/daos/dataset.py

```diff
@@ -150,6 +152,17 @@ def validate_metrics_uniqueness(dataset_id: int, metrics_names: list[str]) -> bo
         ).all()
         return len(dataset_query) == 0
 
+    @staticmethod
+    def validate_column_pdf_is_iso8601(dt_format: str) -> bool:
+        if dt_format in ("epoch_s", "epoch_ms", None):
```

Comment:
See previous comment.

Reply:
Any reason this logic is somewhat duplicated in three places? Can this be refactored into a helper method if neither the model nor the DAO seems appropriate for both cases?

Reply:
The TableColumn insert/update using bulk_insert_mappings or bulk_update_mappings does not trigger the ORM's @validates.
I refactored the duplicated code to perform validation exclusively at the DAO level.